mmv1/products/workstations/api.yaml

```diff
@@ -447,3 +447,114 @@ objects:
+    update_verb: :PATCH
+    update_mask: true
+    min_version: beta
+    description: "A single instance of a developer workstation with its own persistent storage."
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        "Workstations": "https://cloud.google.com/workstations/docs/"
+      api: "https://cloud.google.com/workstations/docs/reference/rest/v1beta/projects.locations.workstationClusters.workstationConfigs.workstations/create"
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: "name"
+        base_url: "{{op_id}}"
+        wait_ms: 1000
+        timeouts: !ruby/object:Api::Timeouts
+          insert_minutes: 30
+          update_minutes: 30
+          delete_minutes: 30
+      result: !ruby/object:Api::OpAsync::Result
+        path: "response"
+      status: !ruby/object:Api::OpAsync::Status
+        path: "done"
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: "error"
+        message: "message"
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: "workstationId"
+        required: true
+        input: true
+        url_param_only: true
+        description: |
+          ID to use for the workstation.
+      - !ruby/object:Api::Type::String
+        name: "workstationConfigId"
+        required: true
+        input: true
+        url_param_only: true
+        description: |
+          The ID of the workstation cluster config.
+      - !ruby/object:Api::Type::String
+        name: "workstationClusterId"
+        required: true
+        input: true
+        url_param_only: true
+        description: |
+          The name of the workstation cluster.
+      - !ruby/object:Api::Type::String
+        name: "location"
+        input: true
+        required: true
+        url_param_only: true
+        description: |
+          The location where the workstation cluster config should reside.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: "name"
+        output: true
+        description: |
+          The name of the cluster resource.
+      - !ruby/object:Api::Type::String
+        name: "uid"
+        output: true
+        description: |
+          The system-generated UID of the resource.
+      - !ruby/object:Api::Type::String
+        name: "displayName"
+        description: |
+          Human-readable name for this resource.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "labels"
+        description: "Client-specified labels that are applied to the resource and that are also propagated to the underlying Compute Engine resources."
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "annotations"
+        description: "Client-specified annotations. This is distinct from labels."
+      - !ruby/object:Api::Type::String
+        name: "etag"
```

Comment:
nit: Let's just omit this- Terraform always expects to overwrite what's present.

Reply:
`etag` is also present on the other workstation resources, should they be removed there as well?

Reply:
Removed etag, still present on the existing workstation resources though.

Reply:
The other etag fields used the special `- !ruby/object:Api::Type::Fingerprint` type, rather than `string`, so they don't appear as a mutable field in the resource (just an output one). Those are fine to keep that way! We'd need to deprecate it and wait for a major version to remove it on `google_workstations_workstation_cluster` too. With this one we could do the same or remove, although I'd personally lean remove for simplicity.

Reply:
Check, I've just removed it for now as you suggested.